src/TestItemRunner.jl

```diff
@@ -77,7 +77,7 @@ function run_testitem(filepath, use_default_usings, package_name, original_code,
     end
 end
 
-function run_tests(path; filter=nothing)
+function run_tests(path; filter=nothing, verbose=false)
```

Comment:
Should we maybe default to `true`? I feel like it's more useful to see the details by default